packages/core/minos-microservice-networks/tests/test_networks/test_decorators/test_api.py

```diff
@@ -16,10 +16,56 @@ def test_rest_command(self):
         decorator = enroute.rest.command(path="tickets/", method="GET")
         self.assertEqual(RestCommandEnrouteDecorator("tickets/", "GET"), decorator)
 
+    def test_rest_command_kwargs(self):
+        decorator = enroute.rest.command(
+            path="tickets/",
+            method="GET",
+            authorized=True,
+            allowed_groups=[
+                "super_admin",
+                "admin",
+            ],
+        )
+        self.assertEqual(
+            RestCommandEnrouteDecorator(
+                "tickets/",
+                "GET",
+                authorized=True,
+                allowed_groups=[
+                    "super_admin",
+                    "admin",
+                ],
+            ),
+            decorator,
+        )
+
     def test_rest_query(self):
         decorator = enroute.rest.query(path="tickets/", method="GET")
         self.assertEqual(RestQueryEnrouteDecorator("tickets/", "GET"), decorator)
 
+    def test_rest_query_kwargs(self):
+        decorator = enroute.rest.query(
+            path="tickets/",
+            method="GET",
+            authorized=True,
+            allowed_groups=[
+                "super_admin",
+                "admin",
+            ],
```

Comment:
I think a simple `foo="bar"` is enough to check `kwargs`.

Reply:
Done!